### DISCUSSION TITLE
nload for Termux

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#19566] [Q&A] [answered]
Can you add nload for Termux packages? I want to monitor bandwidth usage using nload

**Comment by TomJo2000** [comment]
Please use the package request [issue template](https://github.com/termux/termux-packages/issues/new/choose) and provide necessary information such as the website and source repository of the project.

**Reply by TomJo2000** [reply]
For what it's worth, I did find the repology entry for the package and it meets the requirements of the packaging policy.
https://repology.org/project/nload/information

I'm unsure if it works without root permissions though.

**Reply by TomJo2000** [reply]
Hold on actually, this project hasn't been updated since February 2012.
I'll see if it builds, but that's definitely one point of the packaging policy that's not being met.

**Reply by TomJo2000** [reply]
Segfaults anyway.
Welp that's the end of my interest in getting it working on Termux.
I'll have a look if I can find any similar tool that might be better suited.

**Reply by TomJo2000** [reply]
`bmon` should be a suitable alternative.
Plus, its already packaged.
https://github.com/termux/termux-packages/tree/master/packages/bmon

`pkg i bmon` to install it.

**Comment by TomJo2000** [comment, marked as answer]
To recap, so we have all the information in one convenient place for the marked answer.

- Use `bmon` as a substitute
  - `pkg i bmon`
- `nload` hasn't received any development in 12+ years
- `nload` compiles fine, but segfaults on Termux.
